textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/source/SampleDataStore.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * 
+ */
+package edu.uci.ics.textdb.dataflow.source;

```

Comment:
SampleDataStore is a sample class to store the data in the Lucene index. ScanBasedSourceOperator scans this data. So they are actually not related. Ideally we need to put the actual DataStore classes in textdb-storage project, which I assumed we will be doing at later point.